aspnetcore/mvc/razor-pages/filter/sample/PageFilter/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace PageFilter
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateWebHostBuilder(args).Build().Run();
+        }
+
+        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+             //      .UseStartup<StartupSync>();
```

Comment:
Cleanup

Reply:
I formatted the commented out code but I need all 3 startup class references to test the 3 startup classes - depending on what I want to show.

Reply:
Okies